### PR TITLE
bpo-1635741: Port _posixshmem extension module to multiphase initialization

### PR DESCRIPTION
**Pull request by tiran**
Signed-off-by: Christian Heimes <christian@python.org>

https://bugs.python.org/issue1635741

Automerge-Triggered-By: GH:tiran

### CODE REVIEW

**Comment by tiran** [comment]
@corona10 Most trivial port ever

**Review by corona10 — APPROVED** [review]

File: Misc/NEWS.d/next/C API/2020-11-19-15-33-42.bpo-1635741.9tVsZt.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Port _posixshmem extension module to multiphase initialization (:pep:`489`)
```
Comment (vstinner) [inline Misc/NEWS.d/next/C API/2020-11-19-15-33-42.bpo-1635741.9tVsZt.rst:1]: nitpick: would you mind to add a trialing dot? :-)

```suggestion
Port _posixshmem extension module to multiphase initialization (:pep:`489`).
```

**Review by vstinner — COMMENTED** [review]
LGTM.

File: Misc/NEWS.d/next/C API/2020-11-19-15-33-42.bpo-1635741.9tVsZt.rst (hunk 1 above)
Comment (tiran) [reply Misc/NEWS.d/next/C API/2020-11-19-15-33-42.bpo-1635741.9tVsZt.rst:1]: Next time!

**Pull request merged**